x/intertx/keeper/msg_register_account.go

```diff
@@ -0,0 +1,19 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	types "github.com/regen-network/regen-ledger/x/intertx/types/v1"
+)
+
+// RegisterAccount implements the Msg/RegisterAccount interface
+func (k Keeper) RegisterAccount(ctx context.Context, msg *types.MsgRegisterAccount) (*types.MsgRegisterAccountResponse, error) {
+	if err := k.icaControllerKeeper.RegisterInterchainAccount(sdk.UnwrapSDKContext(ctx), msg.ConnectionId, msg.Owner, msg.Version); err != nil {
+		realErr := fmt.Errorf("ICAControllerKeeper.RegisterInterchainAccount: %w", err)
```

Comment:
Any reason for not using sdkerror here?

Reply:
changed to `sdkerrors.Wrap` https://github.com/regen-network/regen-ledger/pull/1477/commits/ad127d462fd53e78d3b8b2e245626282c1465449